services/ui-src/src/components/export/ExportedOverlayModalReportSection.tsx

```diff
@@ -56,15 +56,14 @@ export const ExportedOverlayModalReportSection = ({
         const nextHeadingLevel = `h${nextLevel}`;
 
         return (
-          <EntityStepCard
+          <ExportedEntityStepCard
```

Comment:
Tests for this component now have `Warning: React does not recognize the headingLevel prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase headinglevel instead. If you accidentally passed it from a parent component, remove it from the DOM element.`